fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisLexer.g4

```diff
@@ -322,6 +322,7 @@ KILL: 'KILL';
 LABEL: 'LABEL';
 LARGEINT: 'LARGEINT';
 LAST: 'LAST';
+LAST_QUERY_ID: 'LAST_QUERY_ID';
```

Comment:
i think we should not support use this function without parentheses, so add this to keyword list is not necessary

Reply:
Thanks. updated